napari/utils/_base.py

```diff
@@ -5,22 +5,10 @@
 the translator before the settings manager is created.
 """
 
-from appdirs import user_config_dir
-from packaging import version
-
-from napari._version import __version__
-
-_dev_version_config_dir = (
-    'dev' if version.parse(__version__).is_devrelease else None
-)
+import os
 
+from napari.utils._appdirs import user_config_dir
 
 _FILENAME = "settings.yaml"
-_APPNAME = "Napari"
-_APPAUTHOR = "Napari"
 _DEFAULT_LOCALE = "en"
-_DEFAULT_CONFIG_PATH = (
-    user_config_dir(_APPNAME, _APPAUTHOR, _dev_version_config_dir)
-    + '/'
-    + _FILENAME
-)
+_DEFAULT_CONFIG_PATH = os.path.join(user_config_dir(), _FILENAME)
```

Comment:
Ah, I see you were on top of things: https://github.com/napari/napari/pull/5322#issuecomment-1313618256
❤️